db/seeds/app.js

```diff
@@ -0,0 +1,22 @@
+const express = require("express");
+const { getTopics } = require("../controllers/getTopics.controller");
+//const app = express();
```

Comment:
On line 3, we still need our app declaration in order to make any requests so we have to put that back in, not comment it out. 
Patrik was referring specifically to removing line 5, where we invoke app.use with express.json(). The latter is only needed for post/patch requests when we are sending data (json format) on the req.body, as it is used to parse incoming data. We don't want it here right now because it's not relevant to this ticket